### DISCUSSION TITLE
We are left the coding, please let come together and complete it.

### DISCUSSION THREAD

**Discussion by akweijudesackey003** [akweijudesackey003/Smart-Campus-Navigating-System.#8] [Announcements]
How are going to start the code, I will consult a Computer science programmer to aid me. What about?

**Comment by Mansah1823** [comment]
Okay we are on codes, but we are left with something small.

**Comment by Michael-Akakpo** [comment]
Alright

**Comment by AndamLoisTech721** [comment]
Sure.

**Comment by Derrickheistman** [comment]
the code is ready

**Comment by Derrickheistman** [comment]
[Fundamentals of software Engineerin.txt](https://github.com/user-attachments/files/18371103/Fundamentals.of.software.Engineerin.txt)

this is text form of the code
